packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -369,6 +384,36 @@ class DeviceDomain extends Domain {
     return new Future<Null>.value();
   }
 
+  /// Forward a host port to a device port.
+  Future<Null> forward(Map<String, dynamic> args) async {
```

Comment:
let's return a `{'port': port}` structure, for the host port that's connected to the device port